Agents/PropertyValueEstimationAgent/Dockerfile

```diff
@@ -0,0 +1,134 @@
+    jpsrm install StackClients ../tmp_stack --jar $stack_clients_jar
+
+# Expose port
+EXPOSE 5000
+
+#------------------------------------------------------
+# Debugging image reusing the base
+#------------------------------------------------------
+FROM base as debug
+# Install additional dependencies for debugging
+RUN pip install debugpy
+
+# Set flask app and development environment to allow for hot-reloading
+ENV FLASK_APP=app/${AGENTDIR}:create_app()
+ENV FLASK_DEBUG=1
+
+# Launch application in container
+WORKDIR /app/$AGENTDIR
+ENTRYPOINT python -m debugpy --listen 0.0.0.0:5678 --wait-for-client -m flask run -h 0.0.0.0 -p 5000
+
+
+#------------------------------------------------------
+# Production image reusing the base
+#------------------------------------------------------
+FROM base as prod
+# Install additional dependencies
+RUN pip install gunicorn
+
+# Copy required source code (as code is not attached as volume)
+WORKDIR /app
+COPY ./$AGENTDIR ./$AGENTDIR
+
+# Set the entrypoint
+# Use shell form to enable variable substitution (https://stackoverflow.com/a/37904830)
+ENTRYPOINT gunicorn --bind 0.0.0.0:5000 "${AGENTDIR}:create_app()"
+
+
+#------------------------------------------------------
+# Debug testing image reusing the base
+#------------------------------------------------------
+FROM base as debug_test
+# Install additional dependencies
+RUN apt-get update
+RUN apt install libpq-dev python3-dev gcc -y
+RUN python -m pip install -e .[dev]
+RUN pip install debugpy
+
+# Set flask app and development environment to allow for hot-reloading
+ENV FLASK_APP=app/${AGENTDIR}:create_app()
+ENV FLASK_DEBUG=1
+
+# Copy docker-compose-file to spin up testcontainers (agent code and tests are mounted as volumes)
+COPY ./docker-compose-testcontainers.yml ./docker-compose.yml
+
+# Launch pytest in container (-s flag to enable more output/logging)
+WORKDIR /app
+# Start dockerised agent app as well as pytest
+# (combining commands using & ensures they are started in parallel in background)
+#CMD python -m debugpy --listen 0.0.0.0:5678 --wait-for-client -m flask run -h 0.0.0.0 -p 5000 & pytest -s
+CMD python -m debugpy --listen 0.0.0.0:5678 -m flask run -h 0.0.0.0 -p 5000 & pytest -s
+
+#------------------------------------------------------
+# Testing image reusing the base
+#------------------------------------------------------
+FROM base as test
+# Install additional dependencies
+RUN apt-get update
+RUN apt install libpq-dev python3-dev gcc -y
+RUN python -m pip install -e .[dev]
+RUN pip install gunicorn
+
+# Copy required source code (as code is not attached as volume)
+WORKDIR /app
+COPY ./$AGENTDIR ./$AGENTDIR
+COPY ./tests ./tests
+# Copy docker-compose-file to spin up testcontainers
+COPY ./docker-compose-testcontainers.yml ./docker-compose.yml
+
+# Ensure SPARQL endpoints contain 'docker.host.internal' instead of 'localhost' for dockerised agent tests
```

Comment:
Please change to `host.docker.internal` in the comment, instead of `docker.host.internal`.

Reply:
Addressed by [70bf83d](https://github.com/cambridge-cares/TheWorldAvatar/pull/454/commits/70bf83da2a3c88d28f664d112439d09e18003cba)